doc/user/zebra.rst

```diff
@@ -559,6 +559,153 @@ presence of the entry.
    21     Static       10.125.0.2  IPv4 Explicit Null
 
 
+.. _zebra-srv6:
+
+Segment-Routing SRv6
+====================
+
+Segment-Routing is source routing paradigm that allows
+network operator to encode network intent into the packets.
+SRv6 is an implementation of Segment-Routing
+with application of IPv6 and segment-routing-header.
+
+All routing daemon can use the Segment-Routing base
+framework implemented on zebra to use SRv6 routing mechanism.
+In that case, user must configure initial srv6 setting on
+FRR's cli or frr.conf or zebra.conf. This section shows how
+to configure SRv6 on FRR. Of course SRv6 can be used as standalone,
+and this section also helps that case.
+
+.. index:: show segment-routing srv6 locator [json]
+.. clicmd:: show segment-routing srv6 locator [json]
+
+   This command dump SRv6-locator configured on zebra.
+   SRv6-locator is used to route to the node before performing
+   the SRv6-function. and that works as aggregation of
+   SRv6-function's IDs.
+   Following console log shows two SRv6-locators LOC1 and LOC2.
+   All locators are identified by unique IPv6 prefix.
+   User can get that information as JSON string when ``json``
+   key word at the end of cli is presented.
+
+::
+
+   router# sh segment-routing-ipv6 locator
```

Comment:
the example `segment-routing-ipv6 locator` doesn't match the 'index' string ("segment-routing srv6 locator") ?

Reply:
This is my mistake. I just fix it

Reply:
I fixed it on this force-push.
https://github.com/FRRouting/frr/compare/79546d9705221b02a181489afaec51f3a8674de2..45f5426e90e20791496a47257ce9aaac2a96b879